pipeline/auto-push.cloudbuild.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: gcr.io/cloud-builders/gsutil
```

Comment:
nit: use `google/cloud-sdk` instead -- it gets updated more consistently https://registry.hub.docker.com/r/google/cloud-sdk

Reply:
Sounds good. I was thinking to use a small image. As we are talking about 1min vs. 16s, it should be fine. Will update.

https://screenshot.googleplex.com/qqDZixy5n6WWuR4